plugins/slack_app/app/helpers/slack_app_helper.rb

```diff
@@ -0,0 +1,12 @@
+require 'uri'
+
+module SlackAppHelper
+  def oauth_url(scopes)
+    redir_url = URI.encode url_for(controller: :slack_app, action: :oauth, only_path: false)
```

Comment:
.to_query will handle the encoding ... atm this will be double encoded ... are the tests not catching that ?